premailer/tests/test_cache.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import absolute_import, unicode_literals
+
+import os
+import unittest
+import imp
+
+import cachetools
+
+
+class TestFunctionCache(unittest.TestCase):
+    def tearDown(self):
+        for key in (
+            "PREMAILER_CACHE",
+            "PREMAILER_CACHE_MAXSIZE",
+            "PREMAILER_CACHE_TTL",
+        ):
+            try:
+                del os.environ[key]
+            except KeyError:
+                pass
+
+    def test_unknown_cache(self):
+        os.environ["PREMAILER_CACHE"] = "UNKNOWN"
+
+        with self.assertRaises(Exception) as assert_context:
+            cache_fd = open("premailer/cache.py", "r")
```

Comment:
The use of forward slashes, will that not break developers on Windows? I.e. use `os.path.join('premailer', 'cache.py')` instead.

Reply:
Totally makes sense. Updated.